frontend/src/component/PublicProfile/CustomFormManager.jsx

```diff
@@ -0,0 +1,311 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import { connect } from 'react-redux'
+import { translate } from 'react-i18next'
+import { IconButton, Checkbox } from 'tracim_frontend_lib'
+// import Form from '@rjsf/core'
+import Form from 'react-jsonschema-form-bs4'
+import TextareaRich from './TextareaRich.jsx'
+
+require('./CustomFormManager.styl')
+
+const DisplayTitle = props => {
+  if (!props.label) return null
+
+  return (
+    <div className='DisplayTitle'>
+      {props.label}
+    </div>
+  )
+}
+
+const DisplaySchemaPropertyString = props => {
+  return (
+    <div className='DisplaySchemaPropertyString'>
+      <span
+        className={classnames(
+          'DisplaySchemaPropertyString__label',
+          { noLabel: !props.label }
+        )}
+      >
+        {props.label}
+      </span>
+      <span
+        className='DisplaySchemaPropertyString__value'
+        dangerouslySetInnerHTML={{ __html: props.value }}
+      />
+    </div>
+  )
+}
+
+const DisplaySchemaPropertyBoolean = props => {
+  return (
+    <div className='DisplaySchemaPropertyString'>
+      <span
+        className={classnames(
+          'DisplaySchemaPropertyString__label',
+          { noLabel: !props.label }
+        )}
+      >
+        {props.label}
+      </span>
+      <span className='DisplaySchemaPropertyString__value'>
+        <Checkbox
+          name={props.label}
+          checked={props.value}
+          onClickCheckbox={() => {}}
+          styleLabel={{ top: '8px', cursor: 'default' }}
+          styleCheck={{ top: '-5px' }}
+        />
+      </span>
+    </div>
+  )
+}
+
+const DisplaySchemaArray = props => {
+  if (!props.valueList) return null
+
+  if (props.valueList.every(value => typeof value === 'string')) {
+    return (
+      <DisplaySchemaPropertyString
+        label={props.label}
+        value={props.valueList.join(', ')}
+      />
+    )
+  }
+
+  return [
+    <DisplayTitle
+      label={props.label}
+      key={props.label}
+    />,
+    props.valueList.map((value, i) => {
+      if (!value) return null
+
+      const valueType = typeof value
+
+      if (valueType === 'object') {
+        return (
```

Comment:
```suggestion
      if (typeof value === 'object') {
        return (
```

Reply:
it might be necessary to handle several values of valueType here later
I coded it that way with that in mind and to have more consistent code with DisplaySchemaObject

I'd rather keep it that way

Reply:
if question is asked, I suggest to comment in order to explain *why* you do like this.

Reply:
comment added